tsconfig.json

```diff
@@ -4,9 +4,15 @@
     "module": "commonjs",
     "outDir": "./lib",
     "rootDir": "./src",
+    "sourceMap": true,
+    "noImplicitAny": true,
+    "jsx": "react",
     "strict": true,
     "noUnusedLocals": true,
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true
-  }
+  },
+  "include": [
```

Comment:
Since we specify a `rootDir`, don't think we need this. [Docs](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html) aren't super clear though, so I could well be wrong. Did adding it fix something?